airbrake-ruby.gemspec

```diff
@@ -25,12 +25,12 @@ DESC
   s.files        = ['lib/airbrake-ruby.rb', *Dir.glob('lib/**/*')]
   s.test_files   = Dir.glob('spec/**/*')
 
+  s.required_ruby_version = '>= 2.0'
```

Comment:
I considered not adding this, but given that we remove a few Ruby 1.9 special cases (especially the one about Exception#cause), this will make the library unusable anyway (I think it will be crashing).